src/doc/man/generated_txt/cargo-add.txt

```diff
@@ -5,7 +5,7 @@ NAME
 
 SYNOPSIS
        cargo add [options] crate…
-       cargo add [options] --path path
+       cargo add [options] --path path [--base base]
```

Comment:
`--base` seems uncommon enough that I'm unsure if we should have it in the usage